recipe/meta.yaml

```diff
@@ -9,22 +9,30 @@ source:
   url: http://www.netlib.org/lapack/lapack-{{ version }}.tgz
   sha1: 7220a9bc9c766cb513007f313b804d88f110c228
 
+  # The next lapack version will have a fix for this.
+  patches:
+    - python-test.patch
+
 build:
   number: 0
-  skip: true  # [win]
 
 requirements:
   build:
     - cmake
     - gcc  # [unix]
     - cloog 0.18.0 10  # [unix]
+    - m2w64-toolchain  # [win]
+    - m2-patch  # [win]
```

Comment:
patch shouldn't be necessary.  Conda-build takes care of this one as an internal dependency.  were there errors without it?